docs/apache-airflow/concepts/deferring.rst

```diff
@@ -140,8 +145,6 @@ Triggers can be as complex or as simple as you like provided you keep inside thi
 
 If you are new to writing asynchronous Python, you should be very careful writing your ``run()`` method; Python's async model means that any code that does not correctly ``await`` when it does a blocking operation will block the *entire process*. Airflow will attempt to detect this and warn you in the triggerer logs when it happens, but we strongly suggest you set the variable ``PYTHONASYNCIODEBUG=1`` when you are writing your Trigger to enable extra checks from Python to make sure you're writing non-blocking code. Be especially careful when doing filesystem calls, as if the underlying filesystem is network-backed it may be blocking.
 
-Right now, Triggers are only used up to their first event, as they are only used for resuming deferred tasks (which happens on the first event fired). However, we plan to allow DAGs to be launched from triggers in future, which is where multi-event triggers will be more useful.
```

Comment:
i moved this to a `note` immediately after where the notion of multi-event triggers is introduced.  until i found this paragraph, i thought multi-event triggers were already supported and was confused when i tried to understand how they could be used.